cli/cmdlineparser.cpp

```diff
@@ -513,9 +505,16 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
 
             // print all possible error messages..
             else if (std::strcmp(argv[i], "--errorlist") == 0) {
-                mShowErrorMessages = true;
-                mSettings.xml = true;
+                // TODO: make this an exclusive option
                 mExitAfterPrint = true;
+                mSettings.loadCppcheckCfg();
+                {
+                    XMLErrorMessagesLogger xmlLogger;
+                    std::cout << ErrorMessage::getXMLHeader(mSettings.cppcheckCfgProductName);
+                    CppCheck::getErrorMessages(xmlLogger);
+                    std::cout << ErrorMessage::getXMLFooter() << std::endl;
+                }
+                return true;
```

Comment:
We can return here now since we no longer rely on other options. `--verbose` never had an effect since we print it as XML which always includes the verbose message.